sh/helm-template/helm-template

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env bash
```

Comment:
Remember to add LICENSE headers to all new files

Reply:
Done